README.md

```diff
@@ -353,10 +353,17 @@ see used in the comments on your PR:
   `extensions/cdc-postgres` module
 * `run-s3-tests` - run all tests in the `extensions/s3` module
 * *`run-nightly-tests` - run nightly (slow) tests. WARNING: Use with care as this is a resource consuming task.*
+* `run-sql-only` - run all tests in `hazelcast-sql`, `hazelcast-distribution`, and `extensions/mapstore` modules
```

Comment:
I would remove the word `all` there since we don't run nightly tests by this phrase, right?

Reply:
Right, I should have realized this is what's meant by "all" in the other ones. How about replacing `all` w/ `quick/integration`?

Reply:
Maybe I would rather use something like `default` for now (there's currently a discussion about how the categories should be named, used etc. so let's keep it as abstract name)